rl4j/rl4j-core/src/main/java/org/deeplearning4j/rl4j/util/VideoRecorder.java

```diff
@@ -45,10 +40,7 @@
  *             .build();
  *         recorder.startRecording("myVideo.mp4");
  *         while(...) {
- *             byte[] data = new byte[160*100*3];
- *             // Todo: Fill data
- *             VideoRecorder.VideoFrame frame = recorder.createFrame(data);
- *             // Todo: Apply cropping or resizing to frame
+ *             INDArray chwData = Nd4j.create()
```

Comment:
`frame` is undefined below, shouldn't it changed to `recorder.record(chwData)`?. But maybe we should remove the example altogether and add one in `deeplearning4j-examples` instead?